lib/NormalModule.js

```diff
@@ -384,9 +384,15 @@ class NormalModule extends Module {
 				}
 
 				if (schema) {
+					let name = "Loader";
+					let baseDataPath = "options";
+					let match;
+					if (schema.title && (match = /^(.+) (.+)$/.exec(schema.title))) {
+						[, name, baseDataPath] = match;
+					}
 					validateOptions(schema, options, {
-						name: getCurrentLoaderName(),
-						baseDataPath: "options"
+						name,
+						baseDataPath
```

Comment:
What do you think about change behavior (prefer schema `title` over arguments) in next schema-utils release?

Reply:
I think both behaviors make sense for different use cases.